valhalla/sif/dynamiccost.h

```diff
@@ -233,7 +233,7 @@ class DynamicCost {
         ((disallow_mask & kDisallowEndRestriction) && edge->end_restriction()) ||
         ((disallow_mask & kDisallowSimpleRestriction) && edge->restrictions()) ||
         ((disallow_mask & kDisallowShortcut) && edge->is_shortcut());
-    return accessible && !assumed_restricted;
+    return accessible && !assumed_restricted && (edge->use() != baldr::Use::kConstruction);
```

Comment:
return `false` in `Allowed` method for roads under construction

Reply:
should we really block roads under construction when the person says "ignore_access=true"?